docs/ref/contrib/auth.txt

```diff
@@ -446,6 +446,15 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
     implement them other than returning an empty set of permissions if
     ``obj is not None``.
 
+    .. attribute:: ModelBackend.allow_inactive_users
+
+        Boolean that determines whether or not the
+        backend will reject inactive users. Defaults to ``False``.
+
+        .. versionchanged:: 1.10
```

Comment:
Just put `.. versionchanged:: 1.10` between the `.. attribute::`line and the paragraph.